### ISSUE TITLE
Website is full of clickbait 'download' ads, looks like a scam site

### ISSUE THREAD

**Issue by wisteso** [ultravnc/UltraVNC#191]
I know this is a legitimate product, but both myself and a coworker were on your primary website looking for the actual download link.

We were presented with no less than a dozen 'Download Here' type scam ads, and it was very difficult to find the actual download link which looked legitimate.

If you guys want people to download and use your software, I highly advise providing a download link here on github or fixing your ad situation.

Thank you for the software itself, it's a great tool.

**Comment by RudiDeVos** [comment]
Does anybody know how to exclude this kind of ads.
Disabling ads is not an option, it pay for all our servers , software certificates etc...

**Comment by panreyes** [comment]
I agree with the OP. Users might be confused by ads with "Download" buttons.

It might be helpful to remove the orange "Download" button for the downloads (and having a text saying "Download_ultravnc_15.0.0.0_dev.zip", or at least to put it in the left side, so it feels like part of the content.

**Comment by gwickman** [comment]
The Ads overwhelm your page and make it hard to find the real download link, this is what [this page](https://uvnc.com/downloads/ultravnc/129-download-ultravnc-1231.html) looked like to me, perhaps it will show different ads for others hence including screen grab. Whoever administers the google adsense account should be able to block these types of ads using the blocking & filtering controls

![image](https://github.com/user-attachments/assets/b469fd54-c04e-4e5c-b025-ef58398a56de)

**Comment by RudiDeVos** [comment]
Tried to filter "download" i hope this help.
expenses > 5k/year. I'm working 10-20h a week on the project, that time is also partly payed by advertising

**Comment by bradisbell** [comment]
@RudiDeVos Your efforts are very much appreciated.  I think the expenses part is fairly easy to reduce using some of those options I mentioned above, but yeah I'm not sure that a call for donations would be enough to cover a reasonable pay.  :-(  I'll keep an eye out... occasionally there are grants for open source things that come up.

**Comment by vanzeelandj** [comment]
The website is appalling. I honestly thought some kind of ad-injection attack was taking place.
This has happened twice now when I've tried to talk a client through accessing your software.
The volume of ad/offerwall etc content is profoundly unprofessional and makes me reconsider using the software.

I can't recommend it to clients who might see your website. It will actually make me look bad for directing them to a scam website (in their eyes).

It's good software itself, but the official website has become so overloaded with junk ad and offerwall content that it looks illegitimate.

Very poor form indeed, please address the condition of the website. At this rate you'll kill your own project with the official website's condition.

I'm now going to look for an alternative. I'm sorry but I can't support what you are doing with the official website.
